src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - devices
+feedback:
+  - api
+---
+
+What if I told you websites could communicate with nearby Bluetooth devices
+in a secure and privacy-preserving way? This way, heart rate monitors, singing
+lightbulbs, [turtles] could interact directly with a website.
+
+Until now, the ability to interact with bluetooth devices has been possible
+only for native apps. The Web Bluetooth API aims to change this and brings it
+to web browsers as well.
+
+## Before we start
+
+This article assumes you have some basic knowledge of how Bluetooth Low
+Energy (BLE) and the [Generic Attribute Profile] (GATT) work.
+
+Even though the [Web Bluetooth API specification] is not finalized yet, the spec
+authors are actively looking for enthusiastic developers to try out this API and
+give [feedback on the spec] and [feedback on the implementation].
+
+A subset of the Web Bluetooth API is available in Chrome OS, Chrome for Android
+6.0, Mac (Chrome 56) and Windows 10 (Chrome 70). This means you should be able
+to [request] and [connect to] nearby Bluetooth Low Energy devices,
+[read]/[write] Bluetooth characteristics, [receive GATT Notifications], know
+when a [Bluetooth device gets disconnected], and even [read and write to
+Bluetooth descriptors].
+
+For Linux and earlier versions of Windows, enable the
+`#experimental-web-platform-features` flag in `chrome://flags`.
+
+### Available for Origin Trials
+
+In order to get as much feedback as possible from developers using the Web
+Bluetooth API in the field, we've previously added this feature in Chrome 53 as
+an [origin trial] for Chrome OS, Android, and Mac.
+
+The trial has successfully ended in January 2017.
+
+## Security Requirements
+
+To understand the security tradeoffs, I recommend the [Web Bluetooth Security
+Model] post from Jeffrey Yasskin, a software engineer on the Chrome team,
+working on the Web Bluetooth API specification.
+
+### HTTPS Only
+
+Because this experimental API is a powerful new feature added to the Web, it is
+made available only to [secure contexts]. This means you'll need to build with
+TLS in mind.
+
+### User Gesture Required
+
+As a security feature, discovering Bluetooth devices with
+`navigator.bluetooth.requestDevice` must be triggered by [a user gesture] such
+as a touch or a mouse click. We're talking about listening to
+[`pointerup`], `click`, and `touchend` events.
+
+```js
+button.addEventListener('pointerup', function(event) {
+  // Call navigator.bluetooth.requestDevice
+});
+```
+
+## Get into the Code
+
+The Web Bluetooth API relies heavily on JavaScript [Promises]. If you're not
+familiar with them, check out this great [Promises tutorial]. One more thing,
+`() => {}` are simply ECMAScript 2015 [Arrow functions] -- they have a shorter
+syntax compared to function expressions and lexically bind the `this` value.
```

Comment:
As @jpmedley mentioned in the USB PR review ([comment](https://github.com/GoogleChrome/web.dev/pull/4095/files#r508844564)), I'm not sure we still need to explain promises and arrow functions. Either way, the two articles should be consistent.

Reply:
I've removed the explanation but kept links as developers interested in Bluetooth or USB may not necessarily be web developers.